connector/pom.xml

```diff
@@ -673,6 +673,13 @@
                     </transformationSets>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.antipathy</groupId>
```

Comment:
This is the lib that does that scala formatting. We need to inherit from the parent and use it here so that we can run maven commands just from the connector folder as well.